reana/reana_dev/git.py

```diff
@@ -724,95 +726,92 @@ def git_push(component):  # noqa: D301
 
-    def _commit_and_publish_version_bumps(components):
-        """Create commit and version bump PRs for all specified components."""
+    def _push_to_origin(components):
         for component in components:
-            has_changes = run_command(
-                "git status --porcelain --untracked-files=no | "
-                "grep setup.py || echo",
-                component=component,
-                return_output=True,
+            branch = run_command(
+                "git branch --show-current", component, return_output=True
+            )
+            run_command(
+                f"git push --force origin {branch}", component,
             )
-            if has_changes:
-                branch_name = datetime.datetime.now().strftime("version-bump-%Y%m%d")
-                create_branch = "git checkout -b {}".format(branch_name)
-                run_command(create_branch, component)
-                create_commit = (
-                    "git add setup.py && "
-                    "git commit "
-                    '-m "installation: shared packages version bump"'
-                )
-                run_command(create_commit, component)
-                create_pr = "hub pull-request -p --no-edit && hub pr list -L 1"
-                run_command(create_pr, component)
-            else:
-                click.echo("{} has no changes, skipping.".format(component))
-
-    cluster_components_with_shared_modules = set(
-        COMPONENTS_USING_SHARED_MODULE_DB
-    ).union(set(COMPONENTS_USING_SHARED_MODULE_COMMONS))
+
+    components = select_components(component)
 
     for module in REPO_LIST_SHARED:
-        last_version = _fetch_latest_pypi_version(module)
-        _update_module_in_cluster_components(module, last_version)
-        click.secho(
-            "✅ {module} updated to: {last_version}".format(
-                module=module, last_version=last_version
-            ),
-            bold=True,
-            fg="green",
+        last_version = fetch_latest_pypi_version(module)
+        update_module_in_cluster_components(
+            module,
+            last_version,
+            components_to_update=components,
+            use_latest_known_tag=use_latest_known_tag,
+            force_feature_branch=True,
         )
 
-    if commit_and_publish:
-        click.secho("Creating version bump commits and pull requests...")
-        _commit_and_publish_version_bumps(cluster_components_with_shared_modules)
-        click.secho(
-            "\nVersion bump commits and pull requests created ✨", bold=True, fg="green"
-        )
-        click.secho(
-            "PR list 👉  https://github.com/search?q="
-            "org%3Areanahub+is%3Apr+is%3Aopen&"
-            "unscoped_q=is%3Apr+is%3Aopen",
-            fg="green",
-        )
+    _create_commit_or_amend(components)
+    ctx.invoke(git_diff, component=component)
+    if click.confirm("Do you want to push your feature branches?"):
```

Comment:
Instead of having an interactive question, what about adding a new `--push` CLI option, similarly to the `git-merge` command, and only push when `--push` option is passed?  (By default, it would not be.) Advantage: consistent behaviour of several `git-...` commands improves UX.